dev/com.ibm.ws.webcontainer/bnd.bnd

```diff
@@ -140,6 +140,7 @@ Import-Package: \
     com.ibm.websphere.security, \
     com.ibm.ws.container.service.app.deploy, \
     com.ibm.ws.genericbnf, \
+	com.ibm.ws.webcontainer.speclevel,\
```

Comment:
Since I don't import `com.ibm.ws.webcontainer.speclevel` directly into a class, I needed to add it manually here in the bnd.bnd. 

```WebContainer.class.getClassLoader().getResourceAsStream("com/ibm/ws/webcontainer/speclevel/servletSpecLevel.properties")```

Unless I'm missing something?

Reply:
Right I understand why you had to add it for that package.  The first part of my comment was just to confirm the desired design.  I think we are in agreement here.  If the exporter goes away and a new exporter is installed then the webcontainer bundle will refresh (get a new class loader) and that is by design.

My other comment was really about all the other imports.  But we should not focus on that question for this PR.